devel/time_conv.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import print_function
```

Comment:
Thanks.  I changed the drawFFT function to respect the input bounds rather than automatically recenter.  But I didn't update this script to reflect that.  Fixed now.